src/storage/hummock_sdk/src/version.rs

```diff
@@ -476,10 +473,26 @@ impl HummockVersion {
             state_table_info_delta: Default::default(),
         }
     }
+
+    pub fn split_change_log(mut self) -> (LocalHummockVersion, HashMap<TableId, TableChangeLog>) {
+        let table_change_log = std::mem::take(&mut self.table_change_log);
```

Comment:
Here after `std::mem::take(&mut self.table_change_log)`, the `epochs` in `self.table_change_log` becomes empty. But we still need to use the `epochs` in `LocalHummockVersion`

Reply:
Good catch  !!!